cmd/nri-rabbitmq/main.go

```diff
@@ -201,3 +205,43 @@ func exitOnError(err error) {
 		os.Exit(-1)
 	}
 }
+
+func alivenessTest(rabbitmqIntegration *integration.Integration, vhostTests []*data.VhostTest, clusterName string) {
```

Comment:
i don't see where this new functions are used in the PR, could you give some context 😄

Reply:
Sure, https://github.com/newrelic/nri-rabbitmq/blob/feature/update-sdk/cmd/nri-rabbitmq/main.go#L75-L76 Actually I haven't created these functions ... maybe I moved the from other file...